src/bundle/Resources/views/Security/login.html.twig

```diff
@@ -35,7 +35,12 @@
                                 </div>
 
                                 <input type="hidden" name="_csrf_token" value="{{ csrf_token("authenticate") }}" />
-                                <input type="hidden" name="_target_path" value="{{ url('ezplatform.dashboard') }}" />
+                                {% set targetPath = url('ezplatform.dashboard') %}
+                                {% set sfRedirect = app.session.get('sf_redirect') %}
```

Comment:
Q: Why `sf_redirect` is taken into consideration in resolving target path? 

PS. In SF 3.4.4 `sf_redirect` was moved from session to cookies (see https://github.com/symfony/symfony/pull/25719).

Reply:
A: if i'm not conneced to admin siteaccess and try to go to http://<...>/admin/Media/Files (from my bookmark), i'm redirected to form login.
After connection, i'm redirected to the dashboard, never to page i've previously requested

I've noticed this "issue" when i've dvelop https://packagist.org/packages/edgar/ez-webpush-bundle

i've noticed quote from @webhdx and your info about deprecation, it was definitely not a good idea to manage this redirection in the login twig :P